functions.php

```diff
@@ -38,3 +37,45 @@ function extractDataAndDisplay(array $data):string
     return $text;
 }
 
+function isFormCorrect(string $title, string $artist_name, string $type, string $description): string
+{
+    if (empty($title) || empty($type) || empty($description) || ($artist_name == '0')) {
+
+        return '<div class="message">' . 'Please make sure to fill all the field in the form. Thanks you' . '</div>';
```

Comment:
You're concatenating a string with another string here, please just make this one string.